pyproject.toml

```diff
@@ -16,8 +16,9 @@ langchain-core = ">=0.2.13,<0.4.0"
 psycopg = "^3"
 psycopg-pool = "^3.2.1"
 sqlalchemy = "^2"
-pgvector = "<0.4"
-numpy = ">=1.21"
+pgvector = ">=0.2.5,<0.4"
+numpy = "^1.21"
```

Comment:
* We should try to relax numpy constraint
* We likely need to expand asyncpg constraint (to support slightly older versions) 0.30.0 is less than a year old I think?